<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CF_LineClassDialog</class>
 <widget class="QDialog" name="CF_LineClassDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="LC_comboBox_Input">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ADC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>POWER</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="LC_comboBox_Output">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>POWER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>POWERF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DAC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ADC</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="LC_Text_ClassID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ClassID:###</string>
   </property>
  </widget>
  <widget class="QComboBox" name="LC_comboBox_Index">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>FREQ</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>existed:(ID,Index,Input,Output)</string>
   </property>
  </widget>
  <widget class="QLabel" name="LC_Msg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Msg:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LC_FindClassID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>203</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>if not find you can:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="LC_newLineClass">
      <property name="text">
       <string>new</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="LC_comboBox_variable">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>DAC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TIME</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CF_LineClassDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CF_LineClassDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
